irohad/consensus/yac/cluster_order.hpp

```diff
@@ -26,13 +26,17 @@ namespace iroha {
     namespace yac {
 
       /**
-       * Class provide ordering on cluster for current round
+       * Provider for the validator order for the current consensus round.
+       *
+       * TODO: I don't like the name Cluster. ValidatorOrdering is better.
```

Comment:
The issue is that we did not introduce validators in the system. A neutral name could be `ParticipantOrder`, since we are providing and order of participants in the consensus round.